docs/topics/db/queries.txt

```diff
@@ -904,6 +919,20 @@ object individually) rather than using the bulk
 :meth:`~django.db.models.query.QuerySet.delete` method of a
 :class:`~django.db.models.query.QuerySet`.
 
+.. versionadded:: 1.9
+
+The :meth:`~django.db.models.query.QuerySet.delete` method returns the number
```

Comment:
I don't see why you split up these two notes and repeated the first sentence? 
Also docs are needed in `docs/ref/models/querysets.txt` and a mention in the release notes.